src/measure_theory/constructions/polish.lean

```diff
@@ -680,4 +681,29 @@ begin
   simp only [id.def, image_id'],
 end
 
+omit hγb
+
+/-- The set of points for which a measurable sequence of functions converges is measurable. -/
+@[measurability] lemma measurable_set_exists_tendsto
+  [hγ : opens_measurable_space γ] [encodable ι] {l : filter ι} [hl : l.ne_bot]
```

Comment:
The assumption `hl` is probably not necessary: if the filter is trivial, then all sequences converge to any limit, so the statement becomes trivial.